src/components/Background/index.js

```diff
@@ -51,14 +53,20 @@ export default class Background extends Component {
   };
 
   renderBackground = () => {
-    const { video, image, poster, autoPlay, width } = this.props;
+    const { video, image, autoPlay, width } = this.props;
 
     if (video && width > breakpointMobile) {
       return (
         <video
+          style={{
+            background: `url(${
+              this.props.image.base64
+            }) no-repeat center center fixed`,
+            backgroundSize: "cover",
```

Comment:
I think you can move `background-repeat: no-repeat; background-attachment: fixed; background-position: center; background-size: cover` to css file.

Reply:
Good idea.